src/core/manager.c

```diff
@@ -1956,9 +1956,12 @@ bool manager_unit_file_maybe_loadable_from_cache(Unit *u) {
         if (u->manager->unit_cache_mtime == 0)
```

Comment:
can you add a comment here, too, explaining what precisely this checks?

Reply:
Hmm, it was added by @bluca  in d904afc730268d50502f764dfd55b8cf4906c46f. I didn't think too much about it. But looking at it carefully, I'm not sure if I understand the purpose. @bluca, can you explain? If we remove it, we would be try to reload the unit if if the cache is not initialized. I'm not sure if that could ever happen (because we load the cache quite early), but either way, I don't see a point for the check.

Reply:
If memory serves, there was another pre-existing check somewhere else that skipped using _mtime if it was not initialised, so I replicated it. I don't remember any specific reason apart from this, so I think it can be removed.

Reply:
Oh, I think I see: unit_file_build_name_map() had a check like this, but it is used in the *inverse* sense: we always rebuild the cache if the timestamp == 0. So yeah, it seems appropriate to remove this.